server/queue/queue-api/src/test/java/org/apache/james/queue/api/MailQueueMetricContract.java

```diff
@@ -132,6 +140,7 @@ default void dequeueShouldPublishDequeueTimeMetric(MailQueueMetricExtension.Mail
         verify(testSystem.getSpyDequeuedMailsTimeMetric(), times(2)).stopAndPublish();
     }
 
+    @Disabled("what do we want to measure ?")
```

Comment:
why is it disabled?

Reply:
Oh, well, It's a question I didn't get the answer to. @chibenwa maybe ?

Reply:
the question asked by Matthieu is "what is supposed to measure the measure?". As the operations are now asynchronous, maybe it's useless and we should remove this metric.

Reply:
Agree, it makes little sense.

However, maybe a rate metric makes more sense. We should ensure that we still maintain a counter IMO

Reply:
I added a commit to remove that metric